server/tso/allocator_manager.go

```diff
@@ -127,6 +238,26 @@ func (am *AllocatorManager) GetAllocator(dcLocation string) (Allocator, error) {
 	return allocatorGroup.allocator, nil
 }
 
+// GetAllocators get all allocators with some filters.
+func (am *AllocatorManager) GetAllocators(withGlobal, withLocal, withInitialized bool) []Allocator {
```

Comment:
The params don’t readable, maybe you can change the judgment condition to a series of options like [`server/schedule/opt/healthy.go`](https://github.com/tikv/pd/blob/master/server/schedule/opt/healthy.go#L37-L61).